htlcswitch/link.go

```diff
@@ -253,6 +259,26 @@ func (l *channelLink) htlcManager() {
 	log.Infof("HTLC manager for ChannelPoint(%v) started, "+
 		"bandwidth=%v", l.channel.ChannelPoint(), l.getBandwidth())
 
+	// If the link have been recreated, than we need to sync the states by
+	// sending the channel reestablishment message.
+	if l.cfg.SyncStates {
```

Comment:
In line with my comment above, this conditional should be removed as resynchronization should be performed unconditionally.